package.json

```diff
@@ -15,6 +15,8 @@
     "debug:firefox": "web-test-runner --watch --config web-test-runner-firefox.config.mjs",
     "debug:no-scoped-registries-polyfill": "npm run debug -- --no-scoped-registries-polyfill",
     "debug:webkit": "web-test-runner --watch --config web-test-runner-webkit.config.mjs",
+    "predeploy": "npm run build",
```

Comment:
can you explain the idea here?

Reply:
the `deploy` script calls this `predeploy` thingy :) -- so we can have other stuff here like copy assets and such